hardware_interface/src/fake_components/generic_system.cpp

```diff
@@ -180,13 +193,47 @@ CallbackReturn GenericSystem::on_init(const hardware_interface::HardwareInfo & i
   }
   initialize_storage_vectors(sensor_fake_commands_, sensor_states_, sensor_interfaces_);
 
+  populate_gpio_interfaces();
```

Comment:
why not use the same approach as for "non-standard" interfaces?

Reply:
I will check this.

Reply:
Done.